meilisearch/client.py

```diff
@@ -205,6 +207,30 @@ def index(self, uid: str) -> Index:
             return Index(self.config, uid=uid)
         raise ValueError("The index UID should not be None")
 
+    def multi_search(self, queries: List[Dict[str, Any]]) -> Dict[str, List[Dict[str, Any]]]:
+        """Multi Search in the index.
+
+        Parameters
+        ----------
+        queries:
+            List of dictionaries containing the query searched word(s) and the query parameters
+            https://docs.meilisearch.com/reference/api/search.html#search-in-an-index
+
+        Returns
+        -------
+        results:
+            Dictionary of results for each index with hits, offset, limit, processingTime and initial query
```

Comment:
I don't think you need to specify the content of each result especially because we can have different hashes if the person does the request using `hitsPerPage` and another with `limit`. It will contain different set of keys in the end.